week-2-practise/level-2/2-validation.js

```diff
@@ -4,7 +4,13 @@
 // - is less than or equal to 100
 // Tip: write other small functions for each requirement
 
-function validate(num) {}
+function validate(num) {
+  if (typeof num == "number" && num > 0 && num <= 100) {
```

Comment:
This is correct, but it can be simplified.
```suggestion
  return typeof num == "number" && num > 0 && num <= 100;
```